index.js

```diff
@@ -1,7 +1,7 @@
 const date = new Date();
-document.getElementById('timeandDate').innerHTML = `${date.toDateString()}, ${date.toLocaleTimeString()}`;
+document.getElementById('timeandDate').innerHTML = document.getElementById('timeandDate').innerHTML = `${date.toDateString()}, ${date.toLocaleTimeString()}`;
```

Comment:
it's probably a duplication should be just one assignment.
And about always finding the element via id 5 times: 
In this application it's ok to do it but if you would imagine huge application with a lot of  pages and big html files this operation would slow down your app. Because it would need to find it 5 times and each of those executions will take time.
So in such cases it's better to store html object you want to find in a variable and then use it.
```js
const dateTime = document.getElementById('timeandDate');
dateTime.innerHTML = ...
dateTime.style.fontSize = ...
```
And I think you can put these styles in a css file under some class name and then apply the class completely to the html object. This would be more clear approach.